### PR TITLE
UI : Fix the missing drag icon from Teams Table

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

- Fix the missing drag icon from Teams Table

#
### Type of change :

- [x] Bug fix

#
### Frontend Preview (Screenshots) :
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamHierarchy.tsx [hunk 1]
```diff
@@ -132,6 +134,34 @@ const TeamHierarchy: FC<TeamHierarchyProps> = ({
...
+          />
+        </div>
+      ) : (
+        <>
+          <SVGIcons className="drag-icon" icon={Icons.DRAG} />
+          <div className="expand-cell-empty-icon-container" />
+        </>
+      ),
+    []
+  );
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamHierarchy.tsx:163]: Remove it, we have it at `tableUtils` use that one, with additional prop with drag-icon

**Comment by Ashish8689** [comment]
@chirag-madlani  changes updated!

File: openmetadata-ui/src/main/resources/ui/src/styles/components/table.less [hunk 2]
```diff
@@ -9,6 +9,38 @@
...
+              display: inline-flex;
+            }
+          }
+        }
+        .ant-table-cell-row-hover {
+          .drag-icon {
+            visibility: initial;
+          }
+        }
+      }
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/styles/components/table.less:43]: This will affect all the table, Do we want to update all the tables?

**Comment by Ashish8689** [comment]
@chirag-madlani  updated the PR!

File: openmetadata-ui/src/main/resources/ui/src/styles/components/table.less (hunk 2 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/styles/components/table.less]: @chirag-madlani  remove the unwanted css from td tag

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**